src/components/table/index.js

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  DefaultTable,
+  TableContainer,
+  Tr,
+  Td,
+  Th,
+  ArrowIcon,
+  HeaderContainer,
+  HeaderText,
+} from './components';
+
+const Table = ({
+  dataSource,
+  colNames,
+  headerColor,
+  variant,
+  sortIcon,
+  stripped,
+  hoverable,
+  bordered,
+  ...rest
+}) => {
+  const [obtainedDataSource, setObtainedDataSource] = useState(dataSource);
+  const [ascendingOrder, setAscendingOrder] = useState([]);
+
+  const sort = (key, index, order) => {
+    const newObtainedDataSource = [...obtainedDataSource];
+    if (order === 'ascending') {
```

Comment:
Instead of taking the parameter as a string, we could just take it as a boolean. Then only when comparing `value1` and `value2` we decide the return value based on the type of sort.

Reply:
Yeah bro It seems to be good